### PR TITLE
Patch 5

### PR DESCRIPTION
**Pull request by Jasdemi**

### CODE REVIEW

File: ETC_en.tsv [hunk 1]
```diff
@@ -783,9 +783,9 @@ ETC_20150317_000782	Please see the Toyobo Lumpur if you are worried that the poi
 ETC_20150317_000783	Tipio
 ETC_20150317_000784	The top of the yellow pool of body looks supple, but is quite painful if it is hit.
 ETC_20150317_000785	Rondo
-ETC_20150317_000786	Rondo is not a variant of our world of animals. It is a cross between Makai organisims.
-ETC_20150317_000787	TerraNymph
-ETC_20150317_000788	Many of them are playful, so many of you who are interested in having an adventure with them have to be mindful of their belonginngs.
+ETC_20150317_000786	The Rando is not naturally found in our world, but is a cross between various Makai species.
```
Comment (emailboxu) [inline ETC_en.tsv]: The Rondo

File: ETC_en.tsv [hunk 2]
```diff
@@ -800,14 +800,14 @@ ETC_20150317_000799	Red Panto
 ETC_20150317_000800	Blue Panto
 ETC_20150317_000801	Spots Panto
 ETC_20150317_000802	Panto Fighter
-ETC_20150317_000803	A Panto which has excellent comabt skills, this refers to the strong subspecies in the forest terrain.
+ETC_20150317_000803	A Panto has excellent fighting skills and is one of the strongest subspecies in the forest.
```
Comment (emailboxu) [inline ETC_en.tsv]: A Panto that has excellent ...

File: ETC_en.tsv [hunk 3]
```diff
@@ -831,39 +831,39 @@ ETC_20150317_000830	Skill Wood C
 ETC_20150317_000831	Skill Wood D
 ETC_20150317_000832	Skill Wood E
 ETC_20150317_000833	Mallardu
-ETC_20150317_000834	The flowers around the neck are real flowers rather than a decoration.
+ETC_20150317_000834	The flowers around its necks are actually real flowers rather than just a simple decoration.
```
Comment (emailboxu) [inline ETC_en.tsv]: Does the description refer to a monster/thing that has multiple necks?

File: ETC_en.tsv [hunk 4]
```diff
@@ -1512,13 +1512,13 @@ ETC_20150317_001511	You have exceeded the number of players for the team.
 ETC_20150317_001512	Currently I chose Zone waypoint
 ETC_20150317_001513	You cannot create room as the level does not match the level requirement of the game.
 ETC_20150317_001514	{s18}{ol}{#ffffcc} Reached the maximum number of characters capable of producing {nl} To create a new character, you must delete the existing character
-ETC_20150317_001515	You do no thave enough character points.{nl}You can earn character points through quests or missions.
+ETC_20150317_001515	You do no have enough character points.{nl}You can earn character points through quests or missions.
```
Comment (ceddc) [inline ETC_en.tsv]: Maybe it is :

> You do not have

?

File: ETC_en.tsv [hunk 5]
```diff
@@ -831,39 +831,39 @@ ETC_20150317_000830	Skill Wood C
...
-ETC_20150317_000849	They have a very low status in Makai, they are looking for opportunities to make a big ball here.
+ETC_20150317_000849	Low-ranking Makai, always looking for opportunities to climb up the ladder.
 ETC_20150317_000850	Blue Ogouveve Flag
 ETC_20150317_000851	Odamubara
 ETC_20150317_000852	Alchemist Briquettes
 ETC_20150317_000853	Alchemist Tincture
 ETC_20150317_000854	Nepenthes
 ETC_20150317_000855	Seedmia
-ETC_20150317_000856	Seedmia is very tricky because they are able to send the seed up so far.
+ETC_20150317_000856	The Seedmia are very tricky because they are able to shoot their seeds over exceptional long distances.
```
Comment (emailboxu) [inline ETC_en.tsv]: Seedmia are... exceptionally long distances.

File: ETC_en.tsv [hunk 6]
```diff
@@ -831,39 +831,39 @@ ETC_20150317_000830	Skill Wood C
...
 ETC_20150317_000850	Blue Ogouveve Flag
 ETC_20150317_000851	Odamubara
 ETC_20150317_000852	Alchemist Briquettes
 ETC_20150317_000853	Alchemist Tincture
 ETC_20150317_000854	Nepenthes
 ETC_20150317_000855	Seedmia
-ETC_20150317_000856	Seedmia is very tricky because they are able to send the seed up so far.
+ETC_20150317_000856	The Seedmia are very tricky because they are able to shoot their seeds over exceptional long distances. 
 ETC_20150317_000857	Firent
 ETC_20150317_000858	After the plants disaster occured, this monster rented as many of the trees moved, this was named after the first discoverer.
```
Comment (emailboxu) [inline ETC_en.tsv:858]: I believe the term used by many translators here is "The Great Plant Cataclysm".

**Comment by imcgames** [comment]
Big Update!! Thank you!! :+1:

**Pull request merged**